1.1/spec/18-Annex-C.adoc

```diff
@@ -0,0 +1,70 @@
+=== Annex C 
+=== (informative)
+
+==== C.3 Mappings from Simple Features for SQL
+
+The following table maps the functions and properties from Simple Features for SQL to GeoSPARQL.
+
+[frame=none, grid=none, cols="3,3,3,3,3"]
+|===
+| Simple Features for SQL | GeoSPARQL Equivalent | Since GeoSPARQL | Related Property Available | Since GeoSPARQL
+| 2.1.1.1 Basic Methods on Geometry | | | |
+| Dimension(): Double | N/A | - | http://www.opengis.net/ont/geosparql#dimension[geo:dimension] | 1.0
+| GeometryType(): Integer | Class of geometry instance | 1.0 | N/A | -
+| SRID(): Integer | http://www.opengis.net/def/function/geosparql/getSRID[geof:getSRID] | 1.0 | http://www.opengis.net/ont/geosparql#inSRS[geo:inSRS] | 1.1
```

Comment:
I guess we need to remove inSRS if the group decides to move this property to GeoSPARQL 1.2